README.md

```diff
@@ -2,18 +2,19 @@
 
 # Project Name
```

Comment:
- [ ] Please make sure to either remove or fill the empty sections of the readme, and I suggest adding a screenshot of the application to make it look more professional.